### PR TITLE
Implement OCW news view

### LINKED ISSUE #11: Fetch RSS from OCW News and provide data via JSON API for ocw-www
Part of https://github.com/mitodl/ocw-www/issues/6

### PR DESCRIPTION
**Pull request by noisecapella**
Fixes #11 

This implements a simple API which converts the RSS feed for OCW News to JSON. Not all content from the RSS feed is kept, but there should be enough information to create a widget on the ocw-www front page for https://github.com/mitodl/ocw-www/issues/6

### CODE REVIEW

**Comment by noisecapella** [comment]
@mbertrand Actually I forgot about a few changes I need to make so I'm moving this back to work in progress

**Comment by noisecapella** [comment]
@mbertrand Should be ready now

File: news/util.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    Serialize an RSS feed item
+
+    Args:
+        item (xml.etree.elementTree.Element): An <item>
+
+    Returns:
+        dict: A dictionary representation of the RSS item
+    """
+    obj = {child.tag: {"text": child.text, **child.attrib} for child in item}
+    description = obj["description"]
```
Comment (mbertrand) [inline news/util.py:34]: Some of the item attributes look a bit funky in JSON:

{http://purl.org/dc/elements/1.1/}creator
{http://purl.org/rss/1.0/modules/content/}encoded
{http://wellformedweb.org/CommentAPI/}commentRss
{http://purl.org/rss/1.0/modules/slash/}comments

Maybe the curly braces could be removed for the JSON output?

**Review by mbertrand — COMMENTED** [review]
Looks good, just a couple questions

File: main/urls.py [hunk 2]
```diff
@@ -27,6 +28,7 @@
     path("robots.txt", include("robots.urls")),
     # Example view
     path("", index, name="main-index"),
+    path("api/news/", news, name="ocw-news"),
```
[the remark this answers is not in the document]
Comment (noisecapella) [reply main/urls.py:31]: Sure, I'll make `urls.py`

**Review by mbertrand — APPROVED** [review]
👍

**Pull request merged**